Tasks/DeployVisualStudioTestAgent/DeployTestAgent.ps1

```diff
@@ -96,7 +96,7 @@ try
         Write-Verbose "Environment details $environment"
 
         Write-Verbose "Calling Deploy test agent cmdlet"
```

Comment:
Current E2E runs without update TA. This should be caught by UT, added a story clear test debt for tasks